docs/changelog.rst

```diff
@@ -14,6 +14,8 @@ The full release history can be viewed `at the GitHub yank releases page <https:
 
 Bugfixes
 ^^^^^^^^
+- (PR `#830 <https://github.com/choderalab/perses/pull/830>`_)
+  Added support for resuming simulations from the CLI
```

Comment:
Maybe we can be a bit more specific? As I understand, this resumes simulations in some cases but not all of them, is that correct? Maybe we can specify that a bit.

Reply:
I don't think there are any simulations we can't resume, but I will add that the support is limited/fragile